charts/sentry-apps/values.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
You should add some docs for the configurable values following the README in https://github.com/norwoodj/helm-docs